lib/scala/pkg/src/main/scala/org/enso/pkg/Config.scala

```diff
@@ -114,7 +114,16 @@ case class Config(
 
   /** Converts the configuration into a YAML representation. */
   def toYaml: String = {
-    val node          = implicitly[YamlEncoder[Config]].encode(this)
+    val config: Config = this
+    val noDevEdition: Config =
+      if (
+        config.edition.exists(_.parent.exists(p => p.toString == "0.0.0-dev"))
```

Comment:
We have a constant for that, let's use it instead of hardcoding more usages of magic values into the code.
```suggestion
        config.edition.exists(_.parent.exists(p => p.toString == BuildVersion.defaultDevEnsoVersion()))
```

Reply:
Yes, the string constant is a bit unfortunate.

I have already tried use the `defaultDevEnsoVersion()` method, but the `pkg` library doesn't have dependency on `BuildVersion`. I didn't want to change [weight's of the module](http://wiki.apidesign.org/wiki/HeavyWeight), but I can do it if we believe it is the right thing to do.

Reply:
Ah, I see. I thought that this was available in every of our modules.

I don't think it would be bad for `pkg` depend on `BuildVersion`. But if you prefer to avoid the dependency that also sounds ok.

Reply:
I think adding a dependency on `BuildVersion` is ok.

Reply:
Note that `BuildVersion` is inside the transitive closure of dependencies of `engine-runner` anyway. Moreover, it contains just a single class (or two), so its as lightweight as possible.

Reply:
Replaced in 006fce7